build/azure-pipeline.pre-release.yml

```
@@ -0,0 +1,67 @@
+# Run on a schedule
+trigger: none
+pr: none
+
+schedules:
+  - cron: '0 19 * * 1-5' # At 19:00 (7:00 PM) on every day-of-week from Monday through Friday
+    displayName: Nightly Release Schedule
+    always: true
+    branches:
+      include:
+        - main
+
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: microsoft/vscode-engineering
+      ref: main
+      endpoint: Monaco
+
+extends:
+  template: azure-pipelines/extension/pre-release.yml@templates
+  parameters:
+    buildSteps:
+      - task: NodeTool@0
+        inputs:
+          versionSpec: '14.18.2'
+        displayName: Select Node version
+
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: '3.6'
+          addToPath: true
+          architecture: 'x64'
+        displayName: Select Python version
```

Is this for the minimum version we support? If so then we should probably set this to 3.7 and clarify that. We should also update https://github.com/microsoft/vscode-python/wiki/Python-version-support

```suggestion
      - task: UsePythonVersion@0
        inputs:
          versionSpec: '3.7'
          addToPath: true
          architecture: 'x64'
        displayName: Select minimum Python version
```